.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9].[0-9]+'  # matches to backport branches, e.g. 3.6
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
```

Comment:
For testing, you could temporarily allow this to run on any PR (in general, it may be a good idea to test the build but not publish anyway). Then, use `actions/upload-artifact@v2` to get those dists attached to the workflow so you'd be able to test what you produce while working on this PR.